expertise/execute_expertise.py

```diff
@@ -228,33 +198,35 @@ def execute_expertise(config):
             sparse_value=config['model_params'].get('sparse_value'),
             use_redis=config['model_params'].get('use_redis', False)
         )
-        ens_predictor.set_archives_dataset(archives_dataset)
-        ens_predictor.set_submissions_dataset(submissions_dataset)
+        predictor.set_archives_dataset(archives_dataset)
+        predictor.set_submissions_dataset(submissions_dataset)
         specter_publication_path = Path(config['model_params']['publications_path']).joinpath('pub2vec.jsonl')
         if config['model_params'].get('use_redis', False):
             specter_publication_path = None
-        ens_predictor.embed_publications(
+        predictor.embed_publications(
             specter_publications_path=specter_publication_path,
             mfr_publications_path=None, skip_specter=config['model_params'].get('skip_specter', False)
         )
-        ens_predictor.embed_submissions(
+        predictor.embed_submissions(
             specter_submissions_path=Path(config['model_params']['submissions_path']).joinpath('sub2vec.jsonl'),
             mfr_submissions_path=None, skip_specter=config['model_params'].get('skip_specter', False))
-        ens_predictor.all_scores(
+        predictor.all_scores(
             specter_publications_path=specter_publication_path,
             mfr_publications_path=None,
             specter_submissions_path=Path(config['model_params']['submissions_path']).joinpath('sub2vec.jsonl'),
             mfr_submissions_path=None,
             scores_path=Path(config['model_params']['scores_path']).joinpath(config['name'] + '.csv')
         )
 
-        if config['model_params'].get('sparse_value'):
-            ens_predictor.sparse_scores(
-                scores_path=Path(config['model_params']['scores_path']).joinpath(config['name'] + '_sparse.csv')
-            )
-
     if 'alternate_match_group' in config.keys():
-        aggregate_by_group(config)
+        preliminary_scores = aggregate_by_group(config)
+    else:
+        preliminary_scores = predictor.preliminary_scores
```

Comment:
this else is for scores where "notes vs groups" and "notes vs notes"?

the only place we need to group the sores is for "groups vs groups". Am I right?

Reply:
That's my understanding. @haroldrubio can you confirm?